### PR TITLE
Flights widget updates.

### PR DESCRIPTION
**Pull request by remybach**

### CODE REVIEW

File: app/assets/javascripts/lib/widgets/flights_autocomplete.js [hunk 1]
```diff
@@ -126,6 +126,23 @@ define([
...
+          if (el == "#js-from-city") {
+            this.$fromAirport.val(airport);
+            this.$fromCity.val(city);
+          } else {
+            this.$toAirport.val(airport);
+            this.$toCity.val(city);
+          }
+        }
+      }.bind(this));
+    }.bind(this));
```
Comment (stevebrownlee) [inline app/assets/javascripts/lib/widgets/flights_autocomplete.js]: Does ie8 support `bind()`?

Comment (remybach) [inline app/assets/javascripts/lib/widgets/flights_autocomplete.js]: Nope, but [we use a polyfill](https://github.com/lonelyplanet/rizzo/blob/master/app/assets/javascripts/app_core.js#L16).

**Comment by stevebrownlee** [comment]
Only one question, and that is irrelevant, then I give :+1:

File: app/assets/javascripts/lib/widgets/flights_autocomplete.js (hunk 1 above)
Comment (stevebrownlee) [inline app/assets/javascripts/lib/widgets/flights_autocomplete.js]: Wow, I never knew that.  Completely changes how I'm going to write JS here from now on.

**Pull request merged**